tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,98 +1,2394 @@
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (with: media) {
+      color: red;
+    }
+  }
+}
+@media print {
+  .page {
+    width: 8in;
+
+    @at-root (without: media) {
+      color: red;
+    }
+  }
+}
+
+`;
+
+exports[`charset.css 1`] = `
+@charset "UTF-8";
+@charset "iso-8859-15";
+@charset 'iso-8859-15'; /* Invalid, wrong quoting style used */
+@charset  "UTF-8"; /* Invalid, more than one space */
+@charset "UTF-8"; /* Invalid, there is a character (a space) before the at-rule */
+@charset UTF-8; /* Invalid, without ' or ", the charset is not a CSS <string> */
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@charset "UTF-8";
+@charset "iso-8859-15";
+@charset 'iso-8859-15'; /* Invalid, wrong quoting style used */
+@charset "UTF-8"; /* Invalid, more than one space */
+@charset "UTF-8"; /* Invalid, there is a character (a space) before the at-rule */
+@charset UTF-8; /* Invalid, without ' or ", the charset is not a CSS <string> */
+
+`;
+
+exports[`counter-style.css 1`] = `
+@counter-style identifier {
+    system: cyclic;
+    symbols: "\\1F44D";
+    suffix: " ";
+}
+@counter-style identifier {
+system: cyclic;
+symbols: "\\1F44D";
+suffix: " ";
+}
+@counter-style identifier{
+    system: cyclic;
+    symbols: "\\1F44D";
+    suffix: " ";
+}
+@counter-style  identifier  {
+    system: cyclic;
+    symbols: "\\1F44D";
+    suffix: " ";
+}
+@counter-style
+identifier
+{
+system
+:
+cyclic
+;
+symbols
+:
+"\\1F44D"
+;
+suffix
+:
+" "
+;
+}
+
+@counter-style
+
+identifier
+
+{
+
+system
+
+:
+
+cyclic
+
+;
+
+symbols
+
+
+:
+
+"\\1F44D"
+
+;
+
+suffix
+
+:
+
+" "
+
+;
+
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+
+@counter-style identifier {
+  system: cyclic;
+
+  symbols: "\\1F44D";
+
+  suffix: " ";
+}
+
+`;
+
+exports[`custom-selector.css 1`] = `
+@custom-selector :--heading h1, h2, h3, h4, h5, h6;
+@custom-selector :--heading h1,h2,h3,h4,h5,h6;
+@custom-selector  :--heading  h1  ,  h2  ,  h3  ,  h4  ,  h5  ,  h6  ;
+@custom-selector :--heading
+    h1, h2, h3, h4, h5, h6;
+@custom-selector
+    :--heading
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6;
+@custom-selector
+:--heading
+h1
+,
+h2
+,
+h3
+,
+h4
+,
+h5
+,
+h6
+;
+@custom-selector
+
+:--heading
+
+h1
+
+,
+
+h2
+
+,
+
+h3
+
+,
+
+h4
+
+,
+
+h5
+
+,
+
+h6
+
+;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
```

Comment:
How about:

```scss
@custom-selector :--heading (
  h1,
  h2,
  h3,
  h4,
  h5,
  h6
);
```

Reply:
@j-f1 invalid syntax, spec don't allow add `(` and `)`